.changeset/lucky-drinks-love.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/backend-common': patch
+'@backstage/plugin-catalog': patch
+'@backstage/plugin-catalog-backend': patch
+'@backstage/plugin-catalog-import': patch
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+Consistent naming of parseGitUrl
```

Comment:
No need for changesets here :p

Reply:
oof, ok, pushed without the file again...